### PR TITLE
System Log API: improve sessionId and tx.id info

### PR DESCRIPTION
**Pull request by ryanoblak-okta**
## Description:
System Log API: 
 - Move `authenticationContext.externalSessionId` and `transaction.id` descriptions to "Event Correlation" section.
  - Clarify descriptions.
  - Simplify example verbiage.
  - Replace proper nouns in Event Correlation Example table headers with JSON field paths.

image
image
image

### Resolves:
* [OKTA-177766](https://oktainc.atlassian.net/browse/OKTA-177766)

/cc @analytics-okta

### CODE REVIEW

**Comment by mbegan** [comment]
Looks good to me, much cleaner.

File: _source/_docs/api/resources/system_log.md [hunk 1]
```diff
@@ -547,11 +547,19 @@ Rate limit violations are sent when a rate limit is exceeded.
...
-## Event ID Correlation
+## Event Correlation
 
-Throughout a user's session many requests (transactions) can occur, such as logging into Okta or opening an application. Within a given request many events may be logged. To illustrate this principle, the table below shows 18 events produced from 13 transactions over 6 different sessions, all performed by one user.
+When looking through the System Log, it is often useful to be able to correlate events to understand the thread of events that happened at a particular time.
 
-| External Session ID       | Transaction ID              | Event ID                             | Event Type                                  | Display Message                   |
+The `LogResponse` object offers two identifiers in this respect:
+  - `authenticationContext.externalSessionId`: identifies events that occurred in the same user session.
+  - `transaction.id`: identifies events that occurred together as part of an operation (for example, a request to Okta’s servers).
```
Comment (bobtiernay-okta) [inline _source/_docs/api/resources/system_log.md]: Should we mention the relationship between`transaction.type` and the semantics of `transaction.id`? For example, `WEB` indicates a the transaction is web request where `JOB` indicates processing in an async task? Or maybe we should just beef up the table that describes these and link to it?

Comment (ryanoblak-okta) [reply _source/_docs/api/resources/system_log.md]: Not in this section, as it's focus is explaining event correlation. The [Transaction Object](https://developer.okta.com/docs/api/resources/system_log#transaction-object) section describes those fields:

image

The Transaction Object field descriptions could use some ❤️, however. I'll touch them up as part of this PR.

Comment (ryanoblak-okta) [reply _source/_docs/api/resources/system_log.md]: @bobtiernay-okta It's not apparent to me where a link to the Transaction Object section would live in the Event Correlation section. Did you have a specific place in mind?

Comment (ryanoblak-okta) [reply _source/_docs/api/resources/system_log.md]: I edited the Transaction Object section:

## Before
image

## After
image

**Comment by nbarbettini** [comment]
Hey @ryanoblak-okta, can you retarget this PR to `source` instead of `weekly`?

**Pull request merged**